app/views/exam_templates/view_logs.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "#{@assignment.short_identifier} #{SplitPdfLog.model_name.human.pluralize}" %>
+
+<% @heading_buttons = [
+    { link_text: t('exam_templates.back_to_exam_template_page'),
+      link_path: assignment_exam_templates_path }
+] %>
+
+<% i = 1 %>
+<% previous_incomplete_num_groups = @split_pdf_logs[0].num_groups_in_incomplete %>
```

Comment:
This doesn't seem like the right comparison to make. For now why don't you just detect whether there are any incomplete groups at all.